.github/workflows/pull-request.yml

```diff
@@ -123,6 +123,95 @@ jobs:
           path: './coverage/lcov.info'
           min_coverage: 95.0
 
+  Health-Check:
+    name: Server health check
+    runs-on: ubuntu-latest
+    needs: Test-Application
+    strategy:
+      matrix:
+        node-version: [14.x]
+    services:
+      mongo:
+        image: mongo:4.4
+        options: >-
+          --health-cmd mongo
```

Comment:
Do we need to start mongo again if it was started already when testing the application?
I'm not sure whether the jobs are independent. The less steps we have the faster the testing so we need to verify.